packages/domain/lib/utils/const.dart

```diff
@@ -0,0 +1,7 @@
+class C {
+  C._();
+
+  static const apiPath = 'https://api.trakt.tv';
+  static const apiAnticipatedPath = 'https://api.trakt.tv/movies/anticipated';
+  static const apiTrendingPath = 'https://api.trakt.tv/movies/trending';
+}
```

Comment:
I don't think it's domain logic

Reply:
But it is domain logic because I use it in usecases to pass to my repository